src/pybind/mgr/cephadm/module.py

```diff
@@ -3943,3 +3985,303 @@ def drain_host(self, hostname: str, force: bool = False, keep_conf_keyring: bool
     def trigger_connect_dashboard_rgw(self) -> None:
         self.need_connect_dashboard_rgw = True
         self.event.set()
+
+
+class CephVolumeLvmList:
```

Comment:
while the class itself is fine, I do wonder if this belongs  somewhere other than module.py which is already almost 4000 lines without this being included. We could maybe even give it its own file or create some sort of ceph volume utils file.

Reply:
yeh, that makes sense